GetMessage/handler.ts

```diff
@@ -117,6 +127,47 @@ type IGetMessageHandler = (
 const LegalMessagePattern = t.interface({ legal_data: LegalData });
 type LegalMessagePattern = t.TypeOf<typeof LegalMessagePattern>;
 
+/**
+ * Checks whether the client service can read advanced message info (read_status and payment_Status)
+ */
+export const canReadAdvancedMessageInfo = (
+  message:
+    | ExternalCreatedMessageWithoutContent
```

Comment:
why a duplicated union type?

Reply:
Nice catch, thanks. Changed to `ExternalCreatedMessageWithContent` in [20fae5a](https://github.com/pagopa/io-functions-services/pull/202/commits/20fae5a64590785abc8512c76fda4f681bbab510)